### PR TITLE
[2.4] Removed the no need client custom path

### PR DESCRIPTION
**Pull request by yhwen**
Fixes # .

### Description

Removed the no need client custom path when creating the simulator client. Which caused appending long system PATH.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/private/fed/app/simulator/simulator_runner.py [hunk 1]
```diff
@@ -331,9 +331,9 @@ def create_client(self, client_name):
             client_name, self.args
         )
         self.federated_clients.append(client)
-        app_root = os.path.join(self.simulator_root, "app_" + client_name)
-        app_custom_folder = os.path.join(app_root, "custom")
-        sys.path.append(app_custom_folder)
+        # app_root = os.path.join(self.simulator_root, "app_" + client_name)
+        # app_custom_folder = os.path.join(app_root, "custom")
+        # sys.path.append(app_custom_folder)
```
Comment (YuanTingHsieh) [inline nvflare/private/fed/app/simulator/simulator_runner.py:336]: Where is the other place that we set this custom path for the clients?

Comment (yhwen) [reply nvflare/private/fed/app/simulator/simulator_runner.py]: It's here:

https://github.com/NVIDIA/NVFlare/blob/main/nvflare/private/fed/app/simulator/simulator_worker.py#L149-L151

**Review by YuanTingHsieh — APPROVED** [review]
LGTM

**Pull request merged**